### PR TITLE
Update context after refreshing

### PR DESCRIPTION
**Pull request by cmatheson**
This makes sure that apps respond to changes to the User or claims in the access token.

### CODE REVIEW

File: src/provider.tsx [hunk 1]
```diff
@@ -1,31 +1,64 @@
...
+    redirectUri,
+    children,
+    onRedirectCallback,
+  } = props;
   const [client, setClient] = React.useState<Client>(NOOP_CLIENT);
   const [state, setState] = React.useState(initialState);
 
+  const onRefresh = React.useCallback(
+    ({ user, accessToken, organizationId }: AuthenticationResponse) => {
+      const { role = null, permissions = [] } = getClaims(accessToken);
```
Comment (mthadley) [inline src/provider.tsx:34]: If `permissions` is `undefined`, does that mean anything?

Double-checking that we want to default to any empty array in that case, or whether we should similarly propagate a `null` value like we have here for `role`.

Comment (cmatheson) [reply src/provider.tsx:34]: good question. the way i imagine developers will work with this is stuff like
```
{('edit:posts' in permissions) && <EditButton />}
```

I don't _think_ there is ever a value in checking if permissions has a value first.

Comment (mthadley) [reply src/provider.tsx:34]: Just a heads up that I don't think `in` works with arrays, but if you swap it out with `.includes`, the example still stands! And there you'd appreciate not needing to worry about checking if `permissions` has a value.

**Review by mthadley — APPROVED** [review]

File: src/provider.tsx (hunk 1 above)
Comment (cmatheson) [reply src/provider.tsx:34]: oh yeah whoops!

**Pull request merged**